src/Catalyst.Cli.Tests/UnitTests/Commands/Response/GetPeerInfoResponseTests.cs

```diff
@@ -0,0 +1,60 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using Catalyst.Cli.Commands;
+using Catalyst.Cli.Tests.UnitTests.Helpers;
+using Catalyst.Core.Lib.Extensions;
+using Catalyst.Protocol.Peer;
+using Catalyst.Protocol.Rpc.Node;
+using Microsoft.Reactive.Testing;
+using NSubstitute;
+using Serilog;
+using Xunit;
+
+namespace Catalyst.Cli.Tests.UnitTests.Commands.Response
+{
+    public sealed class GetPeerInfoResponseTests
+    {
+        private readonly TestScheduler _testScheduler = new TestScheduler();
+
+        [Fact]
+        public void GetPeerInfoResponse_Can_Get_Output()
+        {
+            //Arrange
+            var peer = new PeerInfo();
+            var getPeerInfoResponse = new GetPeerInfoResponse();
+            getPeerInfoResponse.PeerInfo.Add(peer);
+
+            var commandContext = TestCommandHelpers.GenerateCliResponseCommandContext(_testScheduler);
+            var getPeerInfoCommand = new GetPeerInfoCommand(commandContext, Substitute.For<ILogger>());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'getPeerInfoCommand'.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4594394)